build.gradle

```diff
@@ -73,4 +75,10 @@ subprojects {
   afterEvaluate {
     tasks.findByName('check').dependsOn('checkstyle')
   }
-}
\ No newline at end of file
+
+  apply plugin: 'net.ltgt.errorprone'
+
+  dependencies {
+    errorprone deps.errorProne
+  }
```

Comment:
I'm guessing this is related to the buildscript only comment.

Reply:
i don't follow. what comment?

Reply:
ah, never mind, read the errorprone README: https://github.com/tbroyer/gradle-errorprone-plugin#usage